### ISSUE TITLE
Ask Alby to integrate it

### ISSUE THREAD

**Issue by ffrediani** [riccardobl/anser-liquid#10]
Hello

Cool add-on for Alby extension. Thanks for developing it.
Did you ask Alby people to look and consider integrate it into their product ? Having Liquid support would be a good thing.

There is a feature request to integrate Liquid already (https://feedback.getalby.com/-alby-extension-request-a-feature/posts/liquid-network-integration-within-the-ui).

**Comment by riccardobl** [comment]
Hello, anser was built for a [bounty by alby and blockstream](https://stacker.news/items/359778) and it was specifically requested to be like this.

Your feature request is more similar to what is done in Aqua wallet, i believe, it should be possible to implement an alby connector that uses boltz (ie. the liquid-lightning swap service used in aqua). 

That said, for my current aqua-like setup i use [lnbits](https://github.com/lnbits/lnbits) with
- this PR for boltz based funding source: https://github.com/lnbits/lnbits/pull/2681
- this extension to accept connections via nwc: https://github.com/lnbits/nwcprovider

and then i can connect alby or any other NWC wallet. 
I don't know if this is an option for you, but if you can run a small server for the lnbits instance, this will give you the aqua experience and more, since you can get a liquid based lnaddress, merchant front etc... using the many extensions available for lnbits.

**Comment by ffrediani** [comment]
Hello @riccardobl thanks for the clarification
You mean Alby personal should merge this code into their existing product by themselves and it is not necessary a pull request ?

**Comment by riccardobl** [comment]
No, i mean this is not a product that can be merged to alby, it is standalone.
But there are better ways to do it alby native
